### PR TITLE
Spectrogram analysis

### PR DESCRIPTION
**Pull request by aryamansriram**
### Summary
The following script creates a spectrogram plot, displays and saves a spectrogram

- Spectrogram is calculated using librosa.
- Plotting is done using matplotlib.

### CODE REVIEW

File: display_spec.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    plt.colorbar(fig, ax=ax)
+    plt.savefig(filepath, bbox_inches='tight', pad_inches=0)
+    plt.show()
+
+if __name__=="__main__":
+    DESCRIPTION = "SPECTROGRAM DISPLAY MODULE"
+    PARSER = argparse.ArgumentParser(description=DESCRIPTION)
+    REQUIRED_ARGUMENTS =  PARSER.add_argument_group("required arguments")
+    REQUIRED_ARGUMENTS.add_argument("-input_file_path",action="store",help="Input filepath to sound file",required=True)
+    REQUIRED_ARGUMENTS.add_argument("-output_file_path",action="store",help="Output file path of location where plot is to be saved",required=True)
```
Comment (nandan-sridhar) [inline display_spec.py:41]: Please make output_file_path optional. If output_file_path is None, then display the spectrogram. If output_file_path is provided then save the figure, but don't display the spectrogram

**Review by nandan-sridhar — COMMENTED** [review]
Please add your EDA to ipython_notebooks folder

File: display_spec.py (hunk 1 above)
Comment (aryamansriram) [reply display_spec.py]: Done

**Review by nandan-sridhar — COMMENTED** [review]
Please remove the files from the EDA_NBs folder. You did a copy instead of a move.

**Comment by nandan-sridhar** [comment]
[Script to display and analyze spectrograms](https://trello.com/c/zquJlXVc/14-script-to-display-and-analyze-spectrograms)

**Pull request merged**